src/tests/ftest/osa/osa_offline_drain.yaml

```diff
@@ -73,12 +73,35 @@ ior:
     iorflags:
       write_flags: "-w -F -k -G 1"
       read_flags: "-F -r -R -k -G 1"
-      ior_api:
-        - DFS
-      obj_class:
-        - RP_2G1
+      api: DFS
+      dfs_oclass: RP_2G1
+      dfs_dir_oclass: RP_2G1
     ior_test_sequence:
     #   - [scmsize, nvmesize, transfersize, blocksize]
     #    The values are set to be in the multiples of 10.
     #    Values are appx GB.
-      - [6000000000, 54000000000, 500000, 500000000]
\ No newline at end of file
+      - [6000000000, 54000000000, 500000, 500000000]
+      - [6000000000, 54000000000, 1000, 500000000]
+mdtest:
+  api: DFS
+  client_processes:
+    np: 30
+  num_of_files_dirs: 4067         # creating total of 120K files
+  test_dir: "/"
+  iteration: 1
+  dfs_destroy: False
+  dfs_oclass: RP_2G1
+  dfs_dir_oclass: RP_2G1
+  manager: "MPICH"
+  flags: "-u"
+  wr_size:
+    32K:
+      write_bytes: 32768
+      read_bytes: 32768
+  verbosity_value: 1
+  depth: 0
+test_obj_class:
+  oclass:
+    - RP_2G8
+    - RP_3G6
+    - RP_4G1
```

Comment:
This is not defined in the test file above.

Reply:
Yes. Added the same template... Will enhance the scripts on future PR test with different object class. I believe it should not affect the functionality for now.

Reply:
Will keep it for now. Will be enhancing the offline drain test in future.